src/core/lib/iomgr/resource_quota.h

```diff
@@ -88,6 +88,8 @@ typedef struct grpc_resource_user grpc_resource_user;
 
 grpc_resource_user *grpc_resource_user_create(
     grpc_resource_quota *resource_quota, const char *name);
+grpc_resource_quota *grpc_resource_user_quota(
```

Comment:
Can you add something like:
// Returns a borrowed reference to the underlying resource quota for this resource user